Gems/OpenXRVk/Code/Source/OpenXRVkInstance.cpp

```diff
@@ -88,26 +85,11 @@ namespace OpenXRVk
 
     AZ::RHI::ResultCode Instance::InitInstanceInternal(AZ::RHI::ValidationMode validationMode)
     {
-#ifdef AZ_PLATFORM_ANDROID
-        // Initialize the loader for Android platform
+#if OPENXRVK_TRAIT_NEEDS_INITIALIZE_XR_LOADER
```

Comment:
Do we need this macro if we have a common implementation of OpenXRInitializeLoader that returns true. Makes the code a little messy.

Reply:
I think it's a larger change doing it that way, plus adding a cpp and a header just to do an empty function seems like a waste, this way it's just a one line macro.